versioned/tiered/mongodb/src/test/java/org/projectnessie/versioned/mongodb/TestMongoDBStore.java

```diff
@@ -28,4 +30,144 @@
   protected long getRandomSeed() {
     return 8612341233543L;
   }
+
+  @Test
```

Comment:
Do you know why these are here when nothing has changed for MongoDB Store? I think the changes to this file should all be removed.

Reply:
This is because I introduced the putConditional and deleteConditional tests to AbstractTestStore in the Rocks ConditionExpression branches. As mongo does not have this implemented in my branch I update TestMongoDBStore to disable these test. I see you have subsequently nested the AbstractTestStore tests and disabled the parent tests for mongo. So reverting this change in my branch brings it in line with that design approach.